lab4/manage_db.py

```diff
@@ -0,0 +1,32 @@
+import db_init
+import plot_data
+from datetime import datetime, timedelta
+import data
+
+
+def add_weekend_rates(currency_data):
+    data_len = len(currency_data)
+    for i in range(data_len):
+        print(type(currency_data[i]['effectiveDate']))
+        prev_date = datetime.strptime(currency_data[i]['effectiveDate'], "%Y-%m-%d")
+        next_date = datetime.strptime(currency_data[i + 1]['effectiveDate'], "%Y-%m-%d")
+        if next_date - prev_date > timedelta(days=1):
+            print(prev_date)
+            print(next_date)
+            currency_data.insert(i + 1, {'effectiveDate': datetime.strftime(prev_date + timedelta(days=1), "%Y-%m-%d"),
```

Comment:
to jest spoko, ale nie zadziała dla 1wszego dnia z zakresu- jeśli to pierwszy dzień jest tym niedostępnym- np. niedzielą czy świętem